tests/fixers/test_on_delete.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+
+from django_upgrade.data import Settings
+from tests.fixers.tools import check_noop, check_transformed
+
+settings = Settings(target_version=(1, 9))
+
+
+@pytest.mark.parametrize(
+    "on_delete_value",
+    [
+        "models.CASCADE",
+        "models.SET_NULL",
+        "models.SET_DEFAULT",
+        "models.PROTECT",
+    ],
+)
+@pytest.mark.parametrize("field_class", ["ForeignKey", "OneToOneField"])
+def test_argument_already_set(on_delete_value, field_class):
+    check_noop(
+        f"""\
+        from django.db import models
+        models.{field_class}("auth.User", on_delete={on_delete_value})
+        """,
+        settings,
+    )
+
+
+@pytest.mark.parametrize("field_class", ["ForeignKey", "OneToOneField"])
+def test_field_class_imported(field_class):
+    check_noop(
+        f"""\
+        from django.db.models import {field_class}
+        {field_class}("auth.User")
+        """,
+        settings,
+    )
```

Comment:
Not supported yet, but maybe this is the 1% case?

Reply:
I think it might be a bit more than 1% of people who like to use this import style. But we can go ahead like so and create an issue to loop back in the future.

Reply:
Issue: #62.